digitalocean/kubernetes/resource_kubernetes_cluster_test.go

```diff
@@ -759,6 +759,29 @@ func TestAccDigitalOceanKubernetesCluster_DestroyAssociated(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanKubernetesCluster_VPCNative(t *testing.T) {
+	rName := acceptance.RandomTestName()
+	var k8s godo.KubernetesCluster
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckDigitalOceanKubernetesClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDigitalOceanKubernetesConfigVPCNative(testClusterVersionLatest, rName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckDigitalOceanKubernetesClusterExists("digitalocean_kubernetes_cluster.foobar", &k8s),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "name", rName),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "region", "nyc1"),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "cluster_subnet", "192.168.0.0/20"),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "service_subnet", "192.168.16.0/24"),
```

Comment:
Can you make the service net a /22? We want to put some limitations in the API to enforce certain minimum sizes.

Reply:
Updated in https://github.com/digitalocean/terraform-provider-digitalocean/pull/1253/commits/c6eced9b8a36303fcfedeeca782e5505b79b6ef4